### DISCUSSION TITLE
Getting RGB images from EuroSAT

### DISCUSSION THREAD

**Discussion by Pangoraw** [torchgeo/torchgeo#592] [General]
Hi!

In some datasets, it seems the bands can be selected when constructing but this not the case for EuroSAT. How can one get the normalized RGB images (similar to [`eurosat/rgb`](https://www.tensorflow.org/datasets/catalog/eurosat#eurosatrgb_default_config) version in tensorflow_datasets) ?

**Comment by Pangoraw** [comment]
From the plotting code, I can do:

```python
img = eurosat[0]["image"]
(img[[3,2,1],:,:] / 3000).clip(0., 1.)
```
But this can still be nice to have the RGB version

**Comment by adamjstewart** [comment]
Related to #289, @nilsleh might be interested in this

**Comment by robmarkcole** [comment]
Cross referencing https://github.com/microsoft/torchgeo/issues/1634

I think an `"rgb_only" = True` parameter would be a useful convenience
